pages/common/pdftk.md

```diff
@@ -0,0 +1,23 @@
+# pdftk
+
+> PDF toolkit.
+
+- Extract range:
```

Comment:
This should be a little more descriptive, e.g. "Extract pages from a pdf file and save them as another file:". ("range" should probably be avoided since the syntax shows that the pages need not be a single continuous interval.)